### PR TITLE
removed references to old colors in latitude

### PR DESCRIPTION
**Pull request by FermiDirak**
This PR replaces all references to `styles/colors` with `./colors`, and removes `styles/colors`.

Monorepo PR: https://github.flexport.io/flexport/flexport/pull/54131

This PR obsoletes https://github.com/flexport/latitude/pull/47

### CODE REVIEW

File: colors.js [hunk 1]
```diff
@@ -71,53 +60,7 @@ export const transmissionColors = Object.freeze({
   green50: "#2DA854",
   green60: "#124D25",
 });
-type ColorMap = {
-  [oldColor: Color]: LatitudeColorValue,
-};
-export const v3ColorMap: ColorMap = Object.freeze({
```
Comment (notandrewkaye) [inline colors.js]: Would we want to keep this around for nuking the deprecated colors?

File: design_system/interfaces/styles/components/Colors.jsx [hunk 2]
```diff
@@ -38,12 +38,12 @@ export default function Colors() {
...
             </div>
             <Highlight languages={["javascript"]}>
-              {`import colors from "styles/colors";`}
+              {`import colors from "latitude/colors";`}
             </Highlight>
             <Tooltip overlay="View our colors on GitHub!" placement="bottom">
               <span>
                 <TextLink
-                  href="https://github.com/flexport/flexport/blob/master/webpack/assets/javascripts/styles/colors.js"
+                  href="https://github.com/flexport/flexport/blob/master/webpack/assets/javascrip../colors.js"
```
Comment (notandrewkaye) [inline design_system/interfaces/styles/components/Colors.jsx]: oops!
```suggestion
                  href="https://github.com/flexport/latitude/blob/master/colors.js"
```
I think that is what you actually want ☝️

File: colors.js (hunk 1 above)
Comment (FermiDirak) [reply colors.js]: Ah, this color mapping is for mapping styles/colors to latitude/colors. Afaik deprecated colors are auxillary to deprecated color constants https://github.com/flexport/latitude/blob/master/styles/deprecatedColorConstants.js

File: design_system/interfaces/styles/components/Colors.jsx (hunk 2 above)
Comment (FermiDirak) [reply design_system/interfaces/styles/components/Colors.jsx]: 🤦‍♂

**Pull request merged**